tests/login-register-logout-smoke.spec.js

```diff
@@ -0,0 +1,27 @@
+const chai = require("chai");
+const expect = chai.expect;
+const App = require("../test/pageobjects/App");
+const LoginLogout = require("../test/pageobjects/LoginLogout");
+
+describe("Login and Logout smoke tests functionality", () => {
+  it("Check Login button when required fields are filled in with valid data", async () => {
```

Comment:
As I see it, you're verifying that user is able to log in, not just the login button.

Reply:
That is what I meant to do.